### PR TITLE
Dockerizing project

### PR DESCRIPTION
**Pull request by Razikus**
This pull request contains few changes:
1. Dockerized frontend and backend
2. Remove build-dependency from frontend (config.js) to download dynamic config on first page open. It can be now changed during runtime (stateless container)
3. Volumes on database and grids
4. Move mapdata.db to relative database/mapdata.db
5. Allow https with traefik and let's encrypt

For docker deploy all you need to do is to fill .env and make 

`docker-compose up -d`

### CODE REVIEW

File: map-frontend/package.json [hunk 1]
```diff
@@ -8,14 +8,16 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "axios": "^0.19.2",
```
Comment (APXEOLOG) [inline map-frontend/package.json]: Please set fixed dependency version (`"axios": "0.19.2"`)

File: map-frontend/package.json [hunk 2]
```diff
@@ -8,14 +8,16 @@
...
     "bootstrap": "4.2.1",
     "bootstrap-select": "1.13.5",
     "jquery": "3.3.1",
     "leaflet": "1.4.0",
     "vue": "2.5.21",
     "vue-resource": "1.5.1",
     "vue-router": "3.0.1",
-    "vue-search-select": "2.8.2"
+    "vue-search-select": "2.8.2",
+    "vuex": "^3.1.3"
```
Comment (APXEOLOG) [inline map-frontend/package.json]: Please set fixed dependency version (`"vuex": "3.1.3"`)

**Review by APXEOLOG — CHANGES_REQUESTED** [review]
Amazing work! Please address small comments and i will merge

File: map-frontend/package.json (hunk 2 above)
Comment (Razikus) [reply map-frontend/package.json]: Fixed in f7d55ab

File: map-frontend/package.json (hunk 1 above)
Comment (Razikus) [reply map-frontend/package.json]: Fixed in f7d55ab

**Pull request merged**